gateleen-hook/src/main/java/org/swisspush/gateleen/hook/HookHandler.java

```diff
@@ -1290,6 +1298,31 @@ private void registerListener(Buffer buffer) {
         listenerRepository.addListener(new Listener(listenerId, getMonitoredUrlSegment(requestUrl), target, hook));
     }
 
+    /**
+     * Parses passed ISO date. The special value EXPIRATION_TIME_NEVER will
+     * result in a date near year 3000 (See: #208).
+     *
+     * @param expirationTimeExpression
+     *      ISO date to parse.
+     * @return
+     *      The parsed date or null if null got passed.
+     * @throws RuntimeException
+     *      In case of problems while parsing the value. This will most
+     *      probably be an {@link IllegalArgumentException}.
+     */
+    private LocalDateTime parseExpirationTime(String expirationTimeExpression ) {
+        LocalDateTime expirationTime;
+        if( EXPIRATION_TIME_NEVER.equals(expirationTimeExpression) ){
+            // No. Not every hook has an expiration time. (See: ISAGD-4839)
```

Comment:
Do not mention Post specific issues like ISAGD-4839. Since this is open source, nobody outside the Post needs this information